index.js

```diff
@@ -52,4 +52,117 @@ const state = {
     }
   ],
   cart: []
-};
\ No newline at end of file
+};
+
+// Element selectors
```

Comment:
I like how you have split your code into different sections such as the selectors, rendering etc.